llama_index/indices/struct_store/json.py

```diff
@@ -0,0 +1,52 @@
+from typing import Any, Optional, Sequence, Union, List, Dict
+
+from llama_index.data_structs.node import Node
+from llama_index.data_structs.table import JSONStructDatapoint
+from llama_index.indices.base_retriever import BaseRetriever
+from llama_index.indices.query.base import BaseQueryEngine
+from llama_index.indices.struct_store.base import BaseGPTStructStoreIndex
+
+JSONType = Union[Dict[str, "JSONType"], List["JSONType"], str, int, float, bool, None]
+
+
+class GPTJSONIndex(BaseGPTStructStoreIndex[JSONStructDatapoint]):
+    index_struct_cls = JSONStructDatapoint
```

Comment:
index_struct_cls is supposed to be a class of an `IndexStruct` (JSONStructDatapoint is not an index struct)

Reply:
Ah ok that may explain the error I'm seeing in the Jupyter notebook too then!